operator/src/test/java/oracle/kubernetes/operator/MainTest.java

```diff
@@ -274,14 +228,14 @@ public void deleteDomainPresenceWithTimeCheck_doNotDelete_with_older_DateTime()
     assertTrue(KubernetesUtils.isFirstNewer(domainMeta, domain2Meta));
   }
 
-  @SuppressWarnings({"unchecked", "SameParameterValue"})
-  private Collection<String> invoke_getDomainNamespaces(String tnValue, String namespace)
-      throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
-    if (getDomainNamespaces == null) {
-      getDomainNamespaces =
-          Main.class.getDeclaredMethod("getDomainNamespacesList", String.class, String.class);
-      getDomainNamespaces.setAccessible(true);
-    }
-    return (Collection<String>) getDomainNamespaces.invoke(null, tnValue, namespace);
+  @Test
+  public void afterReadingExistingResourcesForNamespace_WatcheraAreDefined() {
```

Comment:
It looks like there's a typo for "Watchers" in method name `afterReadingExistingResourcesForNamespace_WatcheraAreDefined`

Reply:
You're correct, and I am sorry I didn't notice it sooner.  I'll try to include it in a future PR.